tests/models/modules/test_SwaVFreezing.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+import torch
+
+from lightly.models.modules.heads import SwaVPrototypes
+
+class TestSwavFreezingPrototypes(unittest.TestCase):
+    def test_swav_frozen_prototypes(self, device: str = "cpu", seed=0):
+        criterion = torch.nn.L1Loss()
+        linear_layer = torch.nn.Linear(8, 8, bias=False)
+        prototypes = SwaVPrototypes(8, 8, 2)
```

Comment:
Please use argument names so it is easier to know what the `8` and `2` belong to.